torch/csrc/inductor/aot_inductor_utils.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <assert.h>
+#include <stdint.h>
+#include <memory>
+#include <stdexcept>
+#include <string>
```

Comment:
This file is very confusing to me. Is this part of the C ABI? Why are there C++ headers???